packages/intent/src/commands/validate.ts

```
@@ -5,10 +5,10 @@ import {
   writeFileSync,
 } from 'node:fs'
 import { basename, dirname, join, relative, resolve } from 'node:path'
-import { fail, isCliFailure } from '../cli-error.js'
-import { printWarnings } from '../cli-support.js'
+import { fail, isCliFailure } from '../shared/cli-error.js'
+import { printWarnings } from './support.js'
```

ESLint

[error] `./support.js` import should occur after import of `../setup/workspace-patterns.js`

(import/order)